### PR TITLE
windows: add win virtio driver supports to related cfgs

### PR DESCRIPTION
**Pull request by nickzhq**
ID: 2081921
Signed-off-by: Houqi (Nick) Zuo <hzuo@redhat.com>

### CODE REVIEW

**Comment by nickzhq** [comment]
@zhencliu @luckyh @YongxueHong  Please take a look, thanks!

**Comment by YongxueHong** [comment]
Hi @nickzhq 
Please delete the string `ID: 2081921` from the commit message since it's not necessary.
Thanks.

File: virttest/shared/cfg/guest-os/Windows/Win10/i386.cfg [hunk 1]
```diff
@@ -18,7 +18,7 @@
             cdrom_unattended = "images/win10-32/autounattend.iso"
     sysprep:
         unattended_file = unattended/win10-32-autounattend.xml
-    balloon_service, balloon_hotplug, balloon_memhp:
+    balloon_service, balloon_hotplug, balloon_memhp, win_virtio_driver_install_by_installer:
```
Comment (zhencliu) [inline virttest/shared/cfg/guest-os/Windows/Win10/i386.cfg:21]: Is it possible that we have a update case, eg. win_virtio_driver_upgrade_by_installer, which will also call balloon services? :)

Comment (nickzhq) [reply virttest/shared/cfg/guest-os/Windows/Win10/i386.cfg:21]: > Is it possible that we have a update case, eg. win_virtio_driver_upgrade_by_installer, which will also call balloon services? :)

Actually, this case is about "update", In this case, the win_virtio_driver_install_by_installer is used as cfg and the win_virtio_driver_update_by_installer is used as py.

**Comment by nickzhq** [comment]
Hello @xiagao , do you think we should add other related cases at the PR?

**Comment by xiagao** [comment]
> add other related case
For now, I think no.

**Comment by nickzhq** [comment]
> Hi @nickzhq Please delete the string `ID: 2081921` from the commit message since it's not necessary. Thanks.

Done

**Review by zhencliu — APPROVED** [review]

**Comment by nickzhq** [comment]
@xiagao @zhencliu @YongxueHong  Please take a look, thanks!

**Comment by zhencliu** [comment]
> @xiagao @zhencliu @YongxueHong Please take a look, thanks!

hi @xiagao @YongxueHong , would you please make a double check? 
This patch is the prerequisite of a tp-qemu patch, so we need to merge this first, thanks.

**Comment by xiagao** [comment]
LGTM, ack.

**Comment by vivianQizhu** [comment]
Code LGTM, but I would suggest to rename the title to "Windows: extend balloon related params to installer case", and add commit message like 'Extend balloon related params to case win_virtio_driver_install_by_installer, as it is updated and adding balloon related function test.'

**Comment by nickzhq** [comment]
> Code LGTM, but I would suggest to rename the title to "Windows: extend balloon related params to installer case", and add commit message like 'Extend balloon related params to case win_virtio_driver_install_by_installer, as it is updated and adding balloon related function test.'

Please check again, thanks!

**Comment by YongxueHong** [comment]
Hi @nickzhq 
Maybe the format of the commit message should update:
```
message         <- Any number of lines, limited to 72 characters per line.
                <- Blank line 
```
The format of the commit, can refer to https://avocado-framework.readthedocs.io/en/98.0/guides/contributor/chapters/styleguides.html#commit-style-guide

**Comment by nickzhq** [comment]
> Hi @nickzhq Maybe the format of the commit message should update:
> 
> ```
> message         <- Any number of lines, limited to 72 characters per line.
>                 <- Blank line 
> ```
> 
> The format of the commit, can refer to https://avocado-framework.readthedocs.io/en/98.0/guides/contributor/chapters/styleguides.html#commit-style-guide

Done, please check again, thanks!

**Review by YongxueHong — APPROVED** [review]
LGTM.

**Pull request merged**